stable/unifi/templates/deployment.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "unifi.fullname" . }}
+  labels:
+    app: {{ template "unifi.name" . }}
+    chart: {{ template "unifi.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "unifi.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "unifi.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: gui
+              containerPort: {{ .Values.guiService.port }}
```

Comment:
The exposed container ports are being modified here but how is the entrypoint informed of the port configuration?

Reply:
The entrypoint being the optional ingress?  That is defined via the `ingress.yaml` here: https://github.com/billimek/charts/blob/unifi/stable/unifi/templates/ingress.yaml#L36-L37

Reply:
This should be corrected now, too.